ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -3284,6 +3299,8 @@ public Partition getPartition(Table tbl, Map<String, String> partSpec,
     }
     org.apache.hadoop.hive.metastore.api.Partition tpart = null;
     try {
+      // TODO: Either create a new getPartitionWithAuthInfo API with request/response format that takes
+      //  ValidWriteIdList and tableId or if this API is no longer required, remove all the references.
```

Comment:
We are aiming to be backwards compatible. Removing an API method should not be a real option. It might be ok, to return a failure for transactional tables. What was our solution for the new API in the other cases? @nrg4878 might have more ideas for this case

Reply:
Just removed this TODO for now, as we can deal with that later.